### PR TITLE
Add armhf linux build to FV

### PR DESCRIPTION
**Pull request by ystefinko**
Add build script. Update gitlab FV pipeline yml.

Relates-To: OLPEDGE-1187

Signed-off-by: Yaroslav Stefinko <ext-yaroslav.stefinko@here.com>

### CODE REVIEW

File: scripts/linux-armhf/fv/gitlab_build_armhf_fv.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# License-Filename: LICENSE
+
+# This script should run on special docker image with armhf packages installed.
+# Compiler setup for ARM
+export CC=arm-linux-gnueabihf-gcc-7
+export CXX=arm-linux-gnueabihf-g++-7
+export LD=arm-linux-gnueabihf-ld
+
+mkdir -p build && cd build
+cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -OLP_SDK_BUILD_EXAMPLES=ON -DOLP_SDK_BUILD_DOC=ON -DBUILD_SHARED_LIBS=ON -DOLP_SDK_ENABLE_TESTING=ON ..
```
Comment (andescu) [inline scripts/linux-armhf/fv/gitlab_build_armhf_fv.sh]: We do not need to enable tests and examples

Comment (ystefinko) [reply scripts/linux-armhf/fv/gitlab_build_armhf_fv.sh]: ok, removed those options. Thanks.

File: scripts/linux-armhf/fv/gitlab_build_armhf_fv.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+export LD=arm-linux-gnueabihf-ld
+
+mkdir -p build && cd build
+cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DOLP_SDK_BUILD_DOC=ON -DBUILD_SHARED_LIBS=ON -OLP_SDK_BUILD_EXAMPLES=OFF -DOLP_SDK_ENABLE_TESTING=OFF ..
+make -j$(nproc)
+make docs
+
+# Architecture check after actual compilation
+find . -name "*-tests" -exec ls '{}' \;
+find . -name "*-tests" -exec /usr/bin/arm-linux-gnueabihf-objdump -f '{}' \;
```
Comment (andescu) [inline scripts/linux-armhf/fv/gitlab_build_armhf_fv.sh]: Are these still valid after removing tests?

Comment (ystefinko) [reply scripts/linux-armhf/fv/gitlab_build_armhf_fv.sh]: Agree. Removing that. Nothing to show in this command now.

**Pull request merged**